server/@types/hardstopCutoffDate.ts

```diff
@@ -0,0 +1,3 @@
+export default class HardStopCutoffDate {
```

Comment:
just need to regenerate the types and expose rather than create this
running: `./generate-licence-api-types.sh` and update `licenceApiClientTypes.ts`